models/Issue_State.js

```diff
@@ -0,0 +1,29 @@
+const { Model, DataTypes } = require('sequelize');
+const sequelize = require('../config/connection');
+
+class Issue_State extends Model {}
+
+Issue_State.init(
+    {
+    // Primary key
+      id: {
+        type: DataTypes.INTEGER,
+        allowNull: false,
+        primaryKey: true,
+        autoIncrement: true
+      },
+      name: {
+        type: DataTypes.STRING,
+        allowNull: true
```

Comment:
We should not allow null

Reply:
name: {
        type: DataTypes.STRING,
        allowNull: false
      }